### PR TITLE
Suggesting Music🎵 based on Facial Emotion predicted

### PR DESCRIPTION
**Pull request by anushavc**
This is my solution to issue #2 . Whenever an emotion is predicted, the user can choose from 5 music options related to that emotion.

### CODE REVIEW

**Review by SanjayMarreddi — CHANGES_REQUESTED** [review]
@anushavc Your work looks good to me.

Just one query. Why did u keep `happymusic5.mp3` file outside the `music` folder of the  `static` folder ? I dont think It is required. Just delete it. Also delete the `jpg Images` from the `static` folders.

Do the above Modifications and `commit` changes to the same `PR`. I will Merge it !

File: README.md [hunk 1]
```diff
@@ -112,6 +112,10 @@ This Project is part of the following programs :
 
 You can find our Code of Conduct [here](/CODE_OF_CONDUCT.md).
 
+## Music
+
+Music: https://www.bensound.com
```
Comment (SanjayMarreddi) [inline README.md]: Also, You may remove the `Music` section from the `README` & put this link of reference in COMMENTS at the corresponding file in the Source Code(May be in the `Visual.html` at the end ), Saying something like `The music is taken from the link : ...`

**Comment by anushavc** [comment]
I have made the suggested changes. Do let me know if it needs any other changes

**Review by SanjayMarreddi — APPROVED** [review]
Everything looks fine. Merging it !

**Comment by SanjayMarreddi** [comment]
@all-contributors please add @anushavc for Code

**Pull request merged**